### DISCUSSION TITLE
Folders inside the app drawer

### DISCUSSION THREAD

**Discussion by sdl820** [LawnchairLauncher/lawnchair#4609] [Ideas]
I think it would improve the overall flow.

**Comment by bgiesing** [comment]
This is now added in Lawnchair 15 Beta 1 but there's still some improvements I personally would like to see missing compared to other launchers like Nova:

1. Being able to add to folder simply by long-pressing an app
2. Sub-folders (Like you could have a Games folder and then folders inside it for different genres like RPG, Puzzle, etc.)
3. Being able to drag out app drawer folders to your homescreen/dock and they stay in sync.
4. Auto-sorting, currently they show up in the order you add them which would be A-Z if you add everything all at once but anything added later gets stuck at the end and you can't drag to re-arrange manually either

**Reply by SuperDragonXD** [reply]
Hello! I know that this reply is a bit late, but for these requests, it's easier to check the existing feature requests (or create a new one) so we can track them much better.
